spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -88,15 +107,15 @@ In the tables below, the placeholder `<name>` should be replaced by the fully qu
 | Counter | None
 | Number of calls prevented from running by an open circuit breaker
 
-|`ft_<name>_circuitbreaker_accumulated_open`
+|`ft_<name>_circuitbreaker_time_open`
```

Comment:
Since we're going to leave the `_total` suffix for counters as Metrics doesn't add it, this should also have the `_total` suffix: `ft_<name>_circuitbreaker_time_open_total`